grepos/app/src/main/java/net/boondockradio/grepos/MainActivity.java

```diff
@@ -22,51 +23,78 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    private final String TAG = "grepos";
+    private final String TAG = MainActivity.class.getSimpleName();
+    private final int LOAD_MORE_THRESHOLD = 1;
+    private final String PER_PAGE = "50";
```

Comment:
normally we use static here

`private static final String PER_PAGE = "50";`

it is because it will never change even across classes, im not sure.
maybe @shaunkawano can tell us why